<launch>

    <node pkg="loam_frame"  type="FrontEndNode"  name="front_end_node"  output="screen" />
    <node pkg="loam_frame"  type="BackEndNode"  name="back_end_node"  output="screen" />
    <node pkg="loam_frame"  type="SavePathNode"  name="save_path_node"  output="screen" />
    <!-- <node pkg="loam_frame"  type="MappingNode"  name="mapping_node"  output="screen" /> -->
    <rosparam file="$(find loam_frame)/config/loam_config.yaml" command="load" />

    <!-- <node pkg="imu_complementary_filter" type="complementary_filter_node" name="complementary_filter_gain_node" output="screen" /> -->

    <node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find loam_frame)/rviz/test.rviz"></node>
    
</launch>
